Main/Peripherals/include/Peripherals/ThrustController.h

```diff
@@ -37,8 +39,11 @@ class ThrustController
 
     FloatPair forwardTrim;
     FloatPair diveTrim;
+    FloatPair diveOffset;
 
     FloatPair getReciprocalValues(float value);
+    float getSafeOffset(float a, float b);
+    void zeroPowerHelper(float &a, float &b);
```

Comment:
Okay, good to know - I'm just very use to references because of the CS teachers.
The zeroing-out is for when the thrusters are turned off. In other words, when you stop the thrusters from diving, there should be no motion. This fragment only executed when the thrusters are set to 0 and not for + or - 0.01F.